types/serializers/wallet.d.ts

```diff
@@ -1,11 +1,13 @@
 import { StringSerializer } from './basic';
 import { StructSerializer, VectorSerializer } from './collections';
 
-export declare const approvalDelta : StructSerializer<{
+declare const approvalDelta : StructSerializer<{
     active_approvals_to_add: VectorSerializer<StringSerializer>,
     active_approvals_to_remove: VectorSerializer<StringSerializer>,
     owner_approvals_to_add: VectorSerializer<StringSerializer>,
     owner_approvals_to_remove: VectorSerializer<StringSerializer>,
     key_approvals_to_add: VectorSerializer<StringSerializer>,
     key_approvals_to_remove: VectorSerializer<StringSerializer>,
 }>;
+
+export default approvalDelta;
```

Comment:
Resolve conflicts more carefully
Use export instead of export default (in `js` file too)